pkg/apis/duck/v1/delivery_types.go

```diff
@@ -81,6 +81,11 @@ type DeliverySpec struct {
 	//
 	// +optional
 	RetryAfterMax *string `json:"retryAfterMax,omitempty"`
+
+	//Format supports more destinations of cloudevents (which may require a specific event format)
```

Comment:
I would change the description here to be more specific

Something like

Format specifies the event format sent to subscriber and dead letter sink. 
Valid values are: 
- `json` <add link_to_cloudevents_spec_json_strucutred_format>
- `binary` <add link_to_cloudevents_spec binary format for HTTP
- `ingress` <add description>